modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -0,0 +1,354 @@
+},
+----
+
+
+You can use these repository URLs to create custom repositories.
+Create a repository for both ``SUSE-MicroOS-5.0-Pool`` and ``SUSE-MicroOS-5.0-Updates``, using the details from the configuration file.
+You also need to create a client tools repository.
+
+
+The details you need for this procedure are:
+
+[[slemicro-repos-manual]]
+[cols="1,1,1,1", options="header"]
+.SLE Micro Custom Repository Settings
+|===
+
+| Option
+| Pool repository
+| Updates repository
+| Client Tools repository
+
+| Repository Label
+| SUSE-MicroOS-5.0-Pool
+| SUSE-MicroOS-5.0-Updates
+| SUSE-MicroOS-5.0-Client-Tools
+
+| Repository URL
+| https://updates.suse.com/SUSE/Products/SUSE-MicroOS/5.0/aarch64/product/?<unique_code>
+| https://updates.suse.com/SUSE/Updates/SUSE-MicroOS/5.0/aarch64/update/?<unique_code>
+| *FIXME* https://download.opensuse.org/repositories/home:/pagarcia:/hackweek20:/SLEMicro5.0-Uyuni-Client-Tools/MicroOS/
+
+| Repository Type
+| yum
+| yum
+| yum
+
+| Has Signed Metadata?
+| Uncheck
+| Uncheck
+| Uncheck
+
+|===
+
+Leave all other fields blank.
+
+include::snippets/manual_repos.adoc[]
+
+
+
+When you have the three repositories created, you can create custom channels.
+
+The details you need for this procedure are:
+
+[[slemicro-channels-custom]]
+[cols="1,1,1,1", options="header"]
+.SLE Micro Custom Channel Settings
+|===
+
+| Option
+| Base Channel
+| Updates repository
+| Client Tools repository
+
+| Channel Name
+| SLE Micro 5.0
+| SLE Micro 5.0 Updates
+| SLE Micro 5.0 Client Tools
+
+| Channel Label
+| suse-microos-5.0-pool
+| suse-microos-5.0-updates
+| suse-microos-5.0-client-tools
+
+| Parent Channel
+| None
+| SLE Micro 5.0
+| SLE Micro 5.0
+
+| Architecture
+| {x86_64}
+| {x86_64}
+| {x86_64}
+
+| Repository Checksum Type
+| sha256
+| sha256
+| sha256
+
+| Channel Summary
+| SUSE MicroOS 5.0 Base Channel
+| SUSE MicroOS 5.0 Updates Channel
+| SUSE MicroOS 5.0 Client Tools Channel
+
+|===
+
+Leave all other fields blank.
+
+include::snippets/manual_channels.adoc[]
+
+include::snippets/manual_associate.adoc[]
+
+////
+
+
+
+== Add Software Channels
+
+Before you register {slemicro} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+ifeval::[{suma-content} == true]
+The products you need for this procedure are:
+
+[[slemicro-channels-wizard]]
+[cols="1,1", options="header"]
+.SLE Micro Products - WebUI
+|===
+
+| OS Version
+| Product Name
+
+| {slemicro} 5.0 {x86_64}
+| SUSE Linux Enterprise Micro 5.0 aarch64
+
+| {slemicro} 5.0 {arm64}
+| SUSE Linux Enterprise Micro 5.0 x86_64
+
+|===
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[sle-channels-cli]]
+[cols="1,1,1", options="header"]
+.SLE Micro Products - CLI
+|===
+
+| OS Version
+| Base Channel
+| Updates Channel
+
+| {slemicro} 5.0 {x86_64}
+| suse-microos-5.0-pool-x86_64
```

Comment:
Assuming the list is correct. Didn't check myself :-)

Reply:
I did :stuck_out_tongue: